core/src/main/java/com/yahoo/imapnio/async/internal/ImapAsyncSessionImpl.java

```diff
@@ -316,6 +326,9 @@ private void requestDoneWithException(@Nonnull final ImapAsyncClientException ca
         // log at error level
         logger.error(SESSION_ERR_REC, sessionId, cause);
         entry.getFuture().done(cause);
+
```

Comment:
please discuss

Reply:
Documented the callers on handleRequestException in following
https://docs.google.com/document/d/1dnXLLnIUBe3-prFLGpo2lLTXsRRudl8USQJF3CXfP1k/edit?usp=sharing